tests/test_app.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+import flask
+import unittest
+from unittest.mock import patch
+from PythonBuddy.app import *
+
+
+class TestIndexPage(unittest.TestCase):
+    def setUp(self):
```

Comment:
Please write comments above each test using the reStructure format (for more info, check out the "reStructured Text Example" in https://realpython.com/documenting-python-code/#docstring-formats)